app:
  ip_address: '127.0.0.1'
  port: 5000
  use_array: true
  robots:
    - "phobos"
    - "deimos"
    - "oberon"
    - "titania"
    - "dione"

map:
  lat: 39.94136
  lon: -75.199492
  zone_number: 18
  zone_id: 'S'
  declination: -11.82 #degrees, minus indicates west
  offset: 196.0
